### PR TITLE
14.00 version 14.0.30035.0

### PR DESCRIPTION
**Pull request by SaiRamNL**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41712)

### CODE REVIEW

File: manifests/1/14/00/14.0.30035.0/14.00.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: 14.00
```
Comment (Trenly) [inline manifests/1/14/00/14.0.30035.0/14.00.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.VCLibs.Desktop.x64
```

File: manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: 14.00
```
Comment (Trenly) [inline manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.VCLibs.Desktop.x64
```

File: manifests/1/14/00/14.0.30035.0/14.00.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: 14.00
```
Comment (Trenly) [inline manifests/1/14/00/14.0.30035.0/14.00.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.VCLibs.Desktop.x64
```

File: manifests/1/14/00/14.0.30035.0/14.00.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: 14.00
+PackageVersion: 14.0.30035.0
+Installers:
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.10042.0
+  Architecture: x64
+  InstallerType: msix
```
Comment (Trenly) [inline manifests/1/14/00/14.0.30035.0/14.00.installer.yaml:11]:
```suggestion
  InstallerType: appx
```

File: manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: 14.00
+PackageVersion: 14.0.30035.0
+PackageLocale: en-US
+Publisher: Microsoft Platform Extensions
+PackageName: Microsoft Visual C++ 2015 UWP Desktop Runtime Package
```
Comment (Trenly) [inline manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml:8]:
```suggestion
PackageName: Microsoft.VCLibs.140.00.UWPDesktop
```

File: manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: 14.00
+PackageVersion: 14.0.30035.0
+PackageLocale: en-US
+Publisher: Microsoft Platform Extensions
```
Comment (Trenly) [inline manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml:7]:
```suggestion
Publisher: Microsoft Corporation
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml (hunk 6 above)
Comment (ItzLevvie) [reply manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml:7]: cc @Trenly: It's supposed to be `Microsoft Platform Extensions`.

```
Name              : Microsoft Visual C++ 2015 UWP Desktop Runtime Package
Publisher         : Microsoft Platform Extensions
Version           : 14.0.30035.0
PackageFamilyName : Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe
Uninstall         : winget uninstall "Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe"
```

File: manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml (hunk 5 above)
Comment (ItzLevvie) [reply manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml:8]: cc @Trenly: It's supposed to be `Microsoft Visual C++ 2015 UWP Desktop Runtime Package`.

```
Name              : Microsoft Visual C++ 2015 UWP Desktop Runtime Package
Publisher         : Microsoft Platform Extensions
Version           : 14.0.30035.0
PackageFamilyName : Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe
Uninstall         : winget uninstall "Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe"
```

File: manifests/1/14/00/14.0.30035.0/14.00.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  InstallerUrl: https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx
```
Comment (ItzLevvie) [inline manifests/1/14/00/14.0.30035.0/14.00.installer.yaml:12]:
```suggestion
  InstallerUrl: https://download.microsoft.com/download/4/7/c/47c6134b-d61f-4024-83bd-b9c9ea951c25/14.0.30035.0-Desktop/Microsoft.VCLibs.x64.14.00.Desktop.appx
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/1/14/00/14.0.30035.0/14.00.installer.yaml (hunk 7 above)
Comment (Trenly) [reply manifests/1/14/00/14.0.30035.0/14.00.installer.yaml:12]: Really? I don’t get any ARP entry when adding it on my VM, and it only shows in Get-AppxPackage after installed

**Pull request closed without merging**